spring-integration-amqp/src/test/java/org/springframework/integration/amqp/outbound/AsyncAmqpGatewayTests.java

```diff
@@ -178,6 +213,7 @@ public void handleMessage(Message<?> message) throws MessagingException {
 		assertEquals("nacknack", ack.getHeaders().get(AmqpHeaders.PUBLISH_CONFIRM_NACK_CAUSE));
 		assertEquals(false, ack.getHeaders().get(AmqpHeaders.PUBLISH_CONFIRM));
 
+		asyncTemplate.stop();
 		receiver.stop();
 	}
```

Comment:
Yes, we should, but it's not critical like leaving containers running; please fix on merge if that's all.